src/commands/RequestDiscordAccessCommand.ts

```diff
@@ -0,0 +1,84 @@
+import { SlashCommandBuilder } from '@discordjs/builders';
+import { CommandInteraction, GuildMember } from 'discord.js';
+import { ExecuteFunction } from '../interfaces/Command';
+import * as students from '../../assets/Listado.json';
```

Comment:
Cuando se importan archivos json se puede omitir el asterisco.

```suggestion
import students from '../../assets/Listado.json';
```